static_report/src/components/shared/Widgets/ColumnAssertionLabelWidget.tsx

```diff
@@ -0,0 +1,49 @@
+import { Flex, Text, Icon } from '@chakra-ui/react';
+import { ReactNode } from 'react';
+import { FiCheck, FiX } from 'react-icons/fi';
+
+import { formatColumnValueWith, formatNumber } from '../../../utils/formatters';
+
+export type ColumnAssertionLabelWidgetProps = {
+  total: number;
+  failed: number;
+  isPassed: boolean;
+  isComparison?: boolean;
+  icon?: ReactNode;
+  comparisonDelta?: ReactNode;
+};
+
+export function ColumnAssertionLabelWidget({
```

Comment:
I'm not sure this should be a widget, as it isn't a middle-level component that reuses our custom lower-level dumber components. 

I suggest this as a candidate for `components/shared/Assertions/..`